lib/xcodeproj/project/object/native_target.rb

```diff
@@ -552,6 +552,36 @@ def add_resources(resource_file_references)
           end
         end
 
+        # remove on demand resource files to the resources build phase of the target.
+        #
+        # @param  [Array<PBXFileReference>] on_demand_resource_file_references
+        #         the files references of the on demand resources to the target.
+        #
+        # @return [void]
+        #
+        def remove_on_demand_resources(on_demand_resource_file_references)
+          on_demand_resource_file_references.each do |file|
```

Comment:
I guess this is already defined? `on_demand_resource_file_references`?

Reply:
nm I am stupid and this is a parameter. Hmmmm this doesn't really match what the function does. Its just removing resource file reference which can be done in `remove_file_reference` inside `build_phase.rb`. I don't think this method is really needed.

Reply:
Or at least can we assert that `ASSET_TAGS` is present on these resources?

Reply:
emmm... Where do I write it more reasonable?